src/tests/file-api-test/README.txt

```diff
@@ -17,22 +17,22 @@ It also tests removing files that are not on the drive.
 
```

Comment:
`(i.e. you cannot open a file for reading/writing without being able to create the file, and you can testing reading/writing without being able to open a file)` change testing to test on line 7.